src/components/Button.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+
+export const FindUsButton = () => {
```

Comment:
These buttons seems the same except for the copy - why make two different versions?

Reply:
They are almost the same. But yeah we could have just pass the content into the button element, but you know. We went the easy route 😉